src/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+
+<body>
+    <section id="main">
+        <h1>Leader boards</h1>
+        <div class="card">
+            <div class="score-container">
+                <div class="recent-scores-title">
+                    <h2 class="recent-title">Recent scores</h2>
+                    <button class="btn-refresh">Refresh</button>
+                </div>
+                <ul class="score-list">
+                    <li class="score-item bg-light">
+                        <p class="score-description">Name : ABC</p>
+                    </li>
+                    <li class="score-item">
+                        <p class="score-description">Name : BCD</p>
+                    </li>
+                    <li class="score-item bg-light">
+                        <p class="score-description">Name : DEF</p>
+                    </li>
+                    <li class="score-item">
+                        <p class="score-description">Name : DEF</p>
```

Comment:
- [ OPTIONAL] Kindly Follow our list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) Be careful with blank lines and indentation.
Do not add indentations without a reason.
  - For readability: add two spaces of indentation. Do not use the tab key